src/keystone/sdk_instructions.txt

```diff
@@ -1,3 +1,5 @@
+Note -- Keystone recommends using Docker because using the pre-made image is usually faster than manually setting things up with below, and it wil enable you to run the Keystone enclave unlike the sdk.
```

Comment:
Small request. Can you rework this file as `instructions.md`, and re-do some of the md-file syntax?

It's a small file, so should not be too difficult to do. (Helps to move all instructions to md-format.)

Reply:
Also, I tried to follow these instructions written as-is, and I don't think they are completely accurate.

I suspect some steps (even including simple ones as `cd` to-this-dir) are perhaps missing.

Can you please work thru these steps and document exactly what one could cut-and-paste, to get the steps to work?